main/libical/APKBUILD

```diff
@@ -38,4 +38,12 @@ package() {
 	make DESTDIR="$pkgdir" install
 }
 
+# libical-dev needs db.h from db-dev, so depend on it
+# /usr/include/libical/icalss.h:492:10: fatal error: db.h: No such file or directory
+dev() {
+	default_dev
+	depends="$depends db-dev"
```

Comment:
`depends="$depends ..."` is problematic in split functions and should be avoided.
See http://lists.alpinelinux.org/alpine-devel/6582.html

Instead do:
````sh
depends_dev="db-dev"
makedepends="$depends_dev ..."
````